README.md

```diff
@@ -0,0 +1,32 @@
+# Flatpak for Xemu
+
+## Installation
+
+1. [Set up Flatpak](https://www.flatpak.org/setup/)
```

Comment:
no user will ever see this.

Reply:
What do you mean? It is mainly for developers, not for flathub end users.

Reply:
May be redundant I agree, but I prefer to have "self-contained" repositories with all the documentation needed to use it, without requiring to use a search engine.